src/test/components/TrackThread.test.js

```diff
@@ -188,4 +190,28 @@ describe('timeline/TrackThread', function() {
       selectionEnd: 2,
     });
   });
+
+  it('does not add disk io markers if none are present', function() {
+    const noMarkers = [];
+    const { container } = setup(getMarkersProfile(noMarkers));
+    expect(queryByTestId(container, 'TimelineMarkersDiskIo')).toBeFalsy();
+  });
+
+  it('adds disk io markers if they are present', function() {
+    const diskIoMarker = [
+      [
+        'create/open',
+        2,
+        {
+          type: 'io',
+          startTime: 2,
+          endTime: 3,
+          source: 'PoisionOIInterposer',
+          filename: '/foo/bar/',
+        },
+      ],
+    ];
+    const { container } = setup(getMarkersProfile(diskIoMarker));
+    expect(queryByTestId(container, 'TimelineMarkersDiskIo')).toBeTruthy();
```

Comment:
I like `getByTestId` because in case it fails the error message is quite good. Also a snapshot could be nice here:
```js
expect(getByTestId('TimelineMarkersDiskIo')).toMatchSnapshot();
```

Reply:
What additional coverage do you feel like a snapshot would provide here? I've been trying to limit snapshots as of late, to where I feel they will be impactful, but I've been getting pretty consistent feedback during reviews to add more. All the components already have existing snapshot tests covering the structure, so the only thing I can see a snapshot doing here is adding to the noise when updating things.